dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -199,6 +203,10 @@ public RestHighLevelClient getClient() {
         return client;
     }
 
+    public NativeHttpRestClient getNativeClient(){
```

Comment:
if we go with the proposed approach we don't need to modify this class

Reply:
yes, we need, because we are just using this new provider for searches, while the default ES client provider will be used for other operations